src/Modules/StatCan.OrchardCore.Hackathon/Services/HackathonService.cs

```diff
@@ -104,58 +114,65 @@ public async Task<ContentItem> JoinTeam(string teamContentItemId, ModelStateDict
                 return null;
             }
 
-            participant.Content.Hacker.Team = JObject.FromObject(new { ContentItemIds = new string[] { team.ContentItemId } });
+            var hacker = user.Properties.ToObject<ContentItem>();
+            hacker.Content.Hacker.Hacker.Team.ContentItemIds.Add(team.ContentItemId);
+            user.Properties = JObject.FromObject(hacker);
```

Comment:
Can you confirm this works ? I would be inclined to think this would override the entire properties of the user.

Reply:
Yeah, I am sure this would override the entire set of properties, not just the hacker properties. 

Something like this:
```
 if (user.Properties.TryGetValue("Hacker", out hackerProperties)){
    var hacker = hackerProperties.ToObject<ContentItem>();
} else {
// never initialized hacker property of the user
//   create the new hacker ContentItem using the _contentManager.New() method
// then assign the property back to the user.
}
```

see: https://github.com/OrchardCMS/OrchardCore/blob/36460afad365703b54bfd35d5d6dc1faa7335c5a/src%2FOrchardCore.Modules%2FOrchardCore.Users%2FDrivers%2FCustomUserSettingsDisplayDriver.cs#L93

Reply:
Also: `user.Properties[contentTypeDefinition.Name] = JObject.FromObject(contentItem);`

Reply:
From what I tested it worked. I just convert all the properties to a ContentItem, add a value and convert it back.

Reply:
Ah I see, I think it's safer to use the `_contentManager.New` in case there are other properties to create from the types